src/index.jsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import ReactGA from 'react-ga';
 
 import ErrorBoundary from './components/error-boundary';
 import App from './components/app';
 
 import './index.pcss';
 
+if (process.env.GA) {
+  ReactGA.initialize(process.env.GA);
+  ReactGA.pageview(window.location.pathname + window.location.search);
+
+  window.onerror = (message, source, lineno, colno, error) => {
```

Comment:
We should be using `addEventListener` I guess? https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#window.addEventListener('error')